gh2/csv.py

```diff
@@ -245,14 +251,39 @@ def main():
         'Milestone'
```

Comment:
There is no column for repo. This means if you generate a file with data from multiple repos there is no easy way to filter out one or more of the repos. I don't know if that is likely to cause an issue.